ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -367,10 +374,13 @@ class BehaviorSpec(NamedTuple):
     - observation_shapes is a List of Tuples of int : Each Tuple corresponds
     to an observation's dimensions. The shape tuples have the same ordering as
     the ordering of the DecisionSteps and TerminalSteps.
+    - sensor_types is a List of SensorTypes, each corresponding to the type of
+    sensor (i.e. observation, goal, etc).
     - action_spec is an ActionSpec NamedTuple
     """
 
     observation_shapes: List[Tuple]
+    sensor_types: List[SensorType]
```

Comment:
Looks like we need to add on ObsSpec tuple to contain sensor types and obs shapes

Reply:
Oh yeah. I was using the old code, but it makes sense to do this when we want to make it official.